dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -219,7 +219,7 @@ def level_properties
     @script_level = ScriptLevelsController.get_script_level(@script, params)
     raise ActiveRecord::RecordNotFound unless @script_level
 
-    @level = @script_level.level
+    @level = select_level
```

Comment:
This will automatically select the current level or sublevel based on the supplied params

Reply:
Is this what fixed the exisiting bug with contained levels that are also sublevels?

Reply:
Nope that was actually just automatically fixed by making `state.progress.currentLevelId` refer to the sublevel ID - the issue was that the ContainedLevelResetButton (the delete button under a contained level submission) uses `state.progress.currentLevelId` to determine if the student has attempted that level: https://github.com/code-dot-org/code-dot-org/blob/5f3acce40a0b26755c9bd762443ea391e2aaa280/apps/src/templates/instructions/ContainedLevelResetButton.jsx#L81-L82
Previously though, when on a sublevel that was a contained level, this ID was referring to the parent level, it was pulling progress from the parent bubble choice level. And the parent level will register an attempt if any of its sublevels have been attempted. So for example if there was a bubble choice levels with two contained sublevels, if a student had attempted the first sublevel and then navigated to the second sublevel, the delete button would be enabled even though they hadn't attempted that level. With this change, when they go to the second sublevel, we will correctly determine that there has not been any attempt made for this sublevel, and the delete button will be disabled.